src/components/Question/Question.js

```diff
@@ -109,26 +119,67 @@ class Question extends Component {
     });
   }
 
+  handleKeyDown(event) {
+    /* don't trigger navigation if the current url isn't a question */
+    const pathname = this.props.history.location.pathname;
+    if (!pathname.match(/\/ch\d\/q\d/)) return false;
+
+    /* assign navigation variables */
+    const ARROW_LEFT = 'ArrowLeft';
+    const ARROW_RIGHT = 'ArrowRight';
+    const { previous, next } = this.state.navigation;
+
+    /* handle navigation */
+    switch (event.key) {
+      case ARROW_LEFT:
+        event.preventDefault();
+        previous.enabled
+          ? this.props.history.push(previous.url)
+          : window.addEventListener(
```

Comment:
as we do this for every case, can't we just put it after the switch?

Reply:
Oh, good catches! I'll push the fixes right away!

Reply:
perhaps you could `return` from the case statements, that way it would only hit the add event listener at the end if we drop through the switch.